test/Tools/circt-bmc/lower-to-bmc.mlir

```diff
@@ -0,0 +1,67 @@
+// RUN: circt-opt --externalize-registers --lower-to-bmc="top-module=comb bound=10" %s | FileCheck %s
```

Comment:
It'd be great if we don't have to run the externalize registers pass in this regression test for lower-bmc. Maybe you could just run it manually and copy-paste the results of it into this file?

Reply:
Yeah, that's a good point! Fixed now!